api/src/pcapi/alembic/versions/20231130T144208_59ecb650a586_change_constraint_for_role_permissions_changed.py

```diff
@@ -0,0 +1,28 @@
+"""
+change constraint for role permissions changed
```

Comment:
Vraiment si tu as la foi, rendre le commentaire plus clair.
Du genre, `Add ROLE_PERMISSIONS_CHANGED to action history constraint`